persistent-sqlite/cbits/config.c

```diff
@@ -4,3 +4,7 @@
 int persistent_sqlite_set_log(void (*logFn)(void*, int, const char*), void* arg) {
   return sqlite3_config(SQLITE_CONFIG_LOG, logFn, arg);
 }
+
+long long int persistent_sqlite3_soft_heap_limit64(long long int N) {
```

Comment:
It's just a layer of protection from future versions of SQlite doing something weird with sqlite3_int64 type. Currently that type is just a typedef for long long it so this function really does nothing. If, however, SQLite ever changes sqlite3_int64 to something different than long long int then this function will either do implicit conversion (if we are sort of lucky) or at least gives us a C compiler error.